docs/js-guide/requirejs-to-webpack.rst

```diff
@@ -0,0 +1,214 @@
+        'case_importer/js/import_history',
+        'case_importer/js/excel_fields',
+        'hqwebapp/js/bootstrap5/widgets',
+        'commcarehq',  // <--- dependency added for webpack
+    ], function (
+        $,
+        _,
+        initialPageData,
+        importHistory,
+        excelFieldsModule
+    ) {
+        ...
+
+
+Bootstrap 3 Entry Points
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the ``domain/js/my_project_settings`` example from Step 1, we can update the ``requirejs_main``
+tag to ``webpack_main_b3``, so that the final usage looks like:
+
+::
+
+    {% webpack_main_b3 "domain/js/my_project_settings" %}
+
+Then, in the file itself, we add the ``commcarehq_b3`` dependency to the list of dependencies:
+
+::
+
+    hqDefine("domain/js/my_project_settings", [
+        'jquery',
+        'knockout',
+        'hqwebapp/js/initial_page_data',
+        'commcarehq_b3',  // <--- dependency added for webpack
+    ], function (
+        $,
+        ko,
+        initialPageData
+    ) {
+        ...
+
+
+Step 3: Verify Webpack Build
+----------------------------
+
+The next step is to ensure that the Webpack build succeeds with the newly-added
+entry point. To do this, restart or run ``yarn dev`` locally. If the build fails,
+it is likely due to a missing alias or application folder path.
+
+If it is a missing alias for a ``yarn`` dependency, first check to if the
+dependency being referenced is using the ``npm`` package name (or ``npm_modules`` filepath)
+or if it referring to an alias previously specified in the ``requirejs_config`` files.
+Ideally, try to use the ``npm`` package/path and update the references for the dependency (if possible).
+If not, you can add the alias in ``webpack/webpack.common.js``.
+
+If an application path is missing, for instance it can't find a path to a dependency
+that is under ``<app_name>/js/<path>``, then it might be that there are no entry points
+in that app yet. If this is the case, add the ``<app_name>`` to the
+``alwaysIncludeApps`` list in ``webpack/generateDetails.js``. Eventually, this list
+won't be necessary once everything is migrated, but for now that's the best workaround.
+
+Once these points are updated, please restart ``yarn dev``. If more build errors persist,
+please see the troubleshooting guide below. If there is no help there, please reach out
+the lead developers in charge of this migration for assistance. Please add troubleshooting
+guidance afterward.
+
+Once the build succeeds, please commit all tho changes for that entry point, with
+a commit message that might look like the following
+
+::
+
+    migrated case_importer/js/main to webpack
+
+
+Step 4: Verify Page Loads Without JavaScript Errors
+---------------------------------------------------
+
+The final step is to ensure that the page with the Entry Point loads without
+JavaScript errors. It is recommended to do this on staging if any additional
+complex local setup is required. Most of the time, entry points should load
```

Comment:
What kind of complex local setup are you thinking about?

Reply:
Say the page in question requires a feature flag or needs data to be available (like a paginated list). maybe i should suggest looking at the QA projects on staging?

Reply:
removing